src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.EolTargetFrameworks.props

```diff
@@ -0,0 +1,25 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.EolTargetFrameworks.props
```

Comment:
I don't think there's any reason to create a separate .props file to list the EOL versions.  Just put it in Microsoft.NET.EolTargetFrameworks.targets.

Reply:
Sounds good, I'll collapse props into targets